collaborative/IOU/heat_index.ipynb

```diff
@@ -162,218 +165,426 @@
   },
   {
    "cell_type": "markdown",
-   "id": "eadfdfab-57fa-4bf4-826e-ec264d97c935",
+   "id": "753706c6-85b7-4965-ae4e-ec884bd24d5f",
    "metadata": {},
    "source": [
-    "Choose a set of months to subset the data by. As we are interested in high heat events, we'll grab the May through September months. You can customize which months you may want to look at by modifying the `month_subset` object in the next cell below. This also helps trim our data size down further, which will speed up future data loading."
+    "Next, load the data into memory. Because we are loading the entire dataset, this will take about 5 minutes, hang tight!"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "011a8667-243d-4767-85a4-f02235aada43",
-   "metadata": {
-    "tags": []
-   },
+   "id": "77dd5ea4-511a-4544-afee-896820f81a89",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "month_subset = [5, 6, 7, 8, 9] # May, June, July, August, September\n",
-    "heatidx_hist_hour = heatidx_hist_hour.isel(time = heatidx_hist_hour.time.dt.month.isin(month_subset))"
+    "heatidx_hist_hour = ck.load(heatidx_hist_hour)"
```

Comment:
Yes! This is something in the backend wrt the heat index calculation itself I think

Reply:
Yes, the warnings can be safely ignored because invalid heat indices resulting from these sqrt operations are replaced with valid values calculated from a different equation.